libraries/botbuilder/tests/teamsInfo.test.js

```diff
@@ -665,7 +665,48 @@ describe('TeamsInfo', () => {
             }
         });
     });
-});
+    describe('getPagedTeamMembers()', () => {
+        it('should throw error when teamId is not present', async () => {
+            try {
+                const context = new TestContext({ type: ActionTypes.message });
+                await TeamsInfo.getPagedTeamMembers(context);
+            } catch(err) {
+                assert.strictEqual(err.message, 'This method is only valid within the scope of a MS Teams Team.');
+            }
+        });
+    });
+    
+    describe('getPagedMembers()', () => {
+        function assertMemberInfo(results, mockedData) {

```

Comment:
where is this function used?

Reply:
It was used to test teamID but I ended up removing the test because me and Joel could not figure out how to get it working..... Should I just remove the function then?

Reply:
yes, if its not being used just remove it